docs/measuring_performance.md

```diff
@@ -5,7 +5,7 @@ option (which will save logs to disk) or the `--track-performance` flag. Both of
 these options will allow you to view the result in `serve` mode by navigating to
 `/$perf`. On that page you will see a timeline something like this:
 
-![example build](/docs/images/example_build.png)
+![example build](images/example_build.png)
```

Comment:
This seems to work fine – and fixes the lint and makes rendering work better in VS code